<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fieldsWizardPage</class>
 <widget class="QWizardPage" name="fieldsWizardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Fields to detail</string>
  </property>
  <property name="subTitle">
   <string>Here you have to add fileds to report</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="4">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Select a page and a detail for fields</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Page:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="pages"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Detail:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="details"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Available</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>On detail</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="5">
      <widget class="QListWidget" name="fields"/>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="addAll">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="report.qrc">
         <normaloff>:/images/arrow-right-double.png</normaloff>:/images/arrow-right-double.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="5">
      <widget class="QListWidget" name="onDetail"/>
     </item>
     <item row="2" column="3">
      <widget class="QToolButton" name="moveUp">
       <property name="text">
        <string>/\</string>
       </property>
       <property name="icon">
        <iconset resource="report.qrc">
         <normaloff>:/images/arrow-up.png</normaloff>:/images/arrow-up.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="add">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="report.qrc">
         <normaloff>:/images/arrow-right.png</normaloff>:/images/arrow-right.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="3" rowspan="3">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>154</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>84</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QToolButton" name="remove">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="report.qrc">
         <normaloff>:/images/arrow-left.png</normaloff>:/images/arrow-left.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QToolButton" name="removeAll">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="report.qrc">
         <normaloff>:/images/arrow-left-double.png</normaloff>:/images/arrow-left-double.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QToolButton" name="moveDown">
       <property name="text">
        <string>\/</string>
       </property>
       <property name="icon">
        <iconset resource="report.qrc">
         <normaloff>:/images/arrow-down.png</normaloff>:/images/arrow-down.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="drawLine">
       <property name="text">
        <string>Add lines between fields</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="report.qrc"/>
 </resources>
 <connections/>
</ui>
